FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -9,6 +9,8 @@
 import sys
```

Comment:
We looked at this, but the command line CLI tools don't share a common base class (at least a locally implemented base class). We felt it would be too intrusive to change them all to do this, although that would be the ideal solution.